data/hai.txt

```diff
@@ -460,6 +460,48 @@ phrase "hostile hai"
 		"space."
 		"territory."
 		"worlds."
+	
+phrase "friendly unfettered"
+	word
+		"We "
+	word
+		"welcome "
+		"accept "
+		"tolerate "
+	word
+		"true"
+		"strong"
+		"ambitious"
+	word
+		" races"
+		" people"
+		" individuals"
```

Comment:
It's preferred to put another "word" line in between that just contains a single space. [Observe.](https://github.com/endless-sky/endless-sky/blob/master/data/hails.txt#L1420)